apps/contentflagging/models.py

```diff
@@ -71,9 +67,8 @@ def flag(self, request, object, flag_type, explanation, recipients=None):
             subject = subject.format(object=object)
             t = loader.get_template('contentflagging/email/flagged.ltxt')
             url = '/admin/contentflagging/contentflag/' + str(object.pk)
-            host = Site.objects.get_current().domain
-            content = t.render(Context({'url':url,'host':host,
-                                        'object':object,'flag_type':flag_type}))
+            content = t.render(Context(
+                {'url': url, 'object': object, 'flag_type': flag_type}))
```

Comment:
Please don't indent just one tab but visually to the next item, so this is correct:

    content = t.render(Context({'url': url,
                                'object': object,
                                'flag_type': flag_type}))